wp-content/plugins/core/src/Nav_Menus/Menu.php

```diff
@@ -6,49 +6,61 @@
 
 class Menu {
 
-	private $args      = [];
-	private $cache_key = '';
+	/**
+	 * @var mixed[]
+	 */
+	private array $args;
+
+	/**
+	 * @var mixed[]
+	 */
+	private array $cache_key = [];
```

Comment:
Because it was never a string to begin with. I can double check this all works correctly though (or write some tests): https://github.com/moderntribe/square-one/blob/ba44b39ecabce2fc4c46be4b99187e55b8644c62/wp-content/plugins/core/src/Nav_Menus/Menu.php#L42-L45

Reply:
Confirmed this is working correctly.

![image](https://user-images.githubusercontent.com/1066195/148471849-0cad2ed8-f97c-40be-9007-5de2087be6c2.png)